microsoft-edge/toc.yml

```diff
@@ -1215,6 +1216,9 @@
             - name: Curation and review process for extensions at Microsoft Edge Add-ons
               href: ./extensions/publish/add-ons-curation.md
 
+            - name: Use the Microsoft Edge Add-ons badge to promote your extension
```

Comment:
where to put Badge article?
* top of toc
* Extensions (or Add-ons) toc bucket & landing page.

What's the scope of the toc bucket and landing page?
* Extensions
* Add-ons

Do we want to try it a couple different ways, to try the resulting UX?